styles/editor.less

```diff
@@ -25,7 +25,6 @@ atom-text-editor, // <- remove when Shadow DOM can't be disabled
   .bracket-matcher .region {
     border-bottom: 1px solid @syntax-cursor-color;
```

Comment:
> From #46: it's very similar to the cursor highlight, and in my peripheral vision, it looks like I have multiple cursors, and makes me hesitate before typing.

Your current changes only remove the background color, but still keep the border color blue:

![screen shot 2015-09-19 at 12 01 08 pm](https://cloud.githubusercontent.com/assets/378023/9974364/88bb9538-5ec6-11e5-9464-3671e873da55.png)

Is that intended? If the blue is confusing, it could be changed to grey by removing the `@syntax-cursor-color` color. Like:

```less
border-bottom: 1px solid;
```